### DISCUSSION TITLE
Already using the original Shairport, Can I install Shairport-Sync?

### DISCUSSION THREAD

**Discussion by ahmadnazirafiq** [mikebrady/shairport-sync#1579] [Q&A]
I have been using the original Shairport (AirPlay 1) on my Pi 4. Is there any proper way to completely remove it and install Shairport-Sync after OR i can install Shairport-Sync on top of it?

**Comment by mikebrady** [comment]
Thanks for the question. The BUILD guide mentions how to install over an existing installation. If you have a firewall running, be aware that AirPlay 2 needs access to more ports. It's also detailed in the guide for NQPTP and in the TROUBLESHOOTING guides.
